node/Cargo.toml

```diff
@@ -63,6 +64,26 @@ jsonrpsee = { version = "0.16.2", features = ["server"] }
 pallet-transaction-payment-rpc = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.39" }
 sc-transaction-pool-api = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.39" }
 
+# Frontier dependencies
+fc-consensus = { git = "https://github.com/paritytech/frontier", branch = "polkadot-v0.9.39" }
+fc-db = { git = "https://github.com/paritytech/frontier", branch = "polkadot-v0.9.39" }
+fc-mapping-sync = { git = "https://github.com/paritytech/frontier", branch = "polkadot-v0.9.39" }
+fc-rpc = { git = "https://github.com/paritytech/frontier", branch = "polkadot-v0.9.39", default-features = false }
+fc-rpc-core = { git = "https://github.com/paritytech/frontier", branch = "polkadot-v0.9.39" }
+fc-storage = { git = "https://github.com/paritytech/frontier", branch = "polkadot-v0.9.39" }
+fp-consensus = { git = "https://github.com/paritytech/frontier", branch = "polkadot-v0.9.39" }
+fp-rpc = { git = "https://github.com/paritytech/frontier", branch = "polkadot-v0.9.39", default-features = false }
+pallet-ethereum = { git = "https://github.com/paritytech/frontier", branch = "polkadot-v0.9.39", default-features = false }
+pallet-evm = { git = "https://github.com/paritytech/frontier", branch = "polkadot-v0.9.39", default-features = false }
+
+# evm-tracing
+moonbeam-primitives-ext = { git = "https://github.com/litentry/astar-frame", branch = "polkadot-v0.9.39" }
+moonbeam-rpc-debug = { git = "https://github.com/litentry/astar-frame", branch = "polkadot-v0.9.39" }
+moonbeam-rpc-primitives-debug = { git = "https://github.com/litentry/astar-frame", branch = "polkadot-v0.9.39", default-features = false }
+moonbeam-rpc-primitives-txpool = { git = "https://github.com/litentry/astar-frame", branch = "polkadot-v0.9.39", default-features = false }
+moonbeam-rpc-trace = { git = "https://github.com/litentry/astar-frame", branch = "polkadot-v0.9.39" }
+moonbeam-rpc-txpool = { git = "https://github.com/litentry/astar-frame", branch = "polkadot-v0.9.39" }
```

Comment:
It's a bit confusing, we have forked repo from `astar-frame` but the pallets are called `moonbeam-*`.

Reply:
This is Astar maintained moonbeam code. Moonbeam does not seperate their code as pallet, Astar maintained one for themself.